karma.auto_test.conf.js

```diff
@@ -0,0 +1,82 @@
+// Karma configuration for TIE.
```

Comment:
Is it possible to just replace the existing Karma config with this one? I don't really see the benefit of having two separate test configs.